src/solver/simulation/economy.cpp

```diff
@@ -71,7 +72,10 @@ AdequacyPatchOptimization::AdequacyPatchOptimization(PROBLEME_HEBDO** problemesH
  EconomyWeeklyOptimization(problemesHebdo)
 {
 }
-void AdequacyPatchOptimization::solve(Variable::State& state, int hourInTheYear, uint numSpace)
+void AdequacyPatchOptimization::solve(Variable::State& state,
```

Comment:
In **AdequacyPatchOptimization::solve(...)**, we should highlight the 3 steps of adq patch first step (local matching + classic optimization + curtailment sharing). We should have : 
```C++
void AdequacyPatchOptimization::solve(...)
{
     this->computeLocalMatchingRules();  // Step 1
     this->solveClassicOptimization(); // Step 2
     // Step 3
     this->prepareCurtailmentSharing(); // Actually computing DTG margin
     this->computeCurtailmentSharing();
}
```